src/resources/views/crud/columns/summernote.blade.php

```diff
@@ -0,0 +1,4 @@
+@php
+    $column['escaped'] = $column['escaped'] ?? false;
+@endphp
+@include('crud::columns.text')
```

Comment:
Isn't it more appropriate for this to load `textarea`, not `text`? Then we don't have to set `escaped` to `false` either.

Reply:
Hmmm this isn't resolved - I see it's still pointing to `text` not `textarea`

Reply:
changed to `textarea`